example/android/app/build.gradle

```diff
@@ -93,6 +93,6 @@ dependencies {
     implementation 'com.google.android.gms:play-services-auth:20.2.0'
     implementation 'com.google.android.gms:play-services-auth-api-phone:18.0.1'
 
-    implementation fileTree(include: ['*.aar'], dir: '/Users/shibu.prasad/Desktop/FlutterStandardWithTPV/')
+    //implementation fileTree(include: ['*.aar'], dir: '/Users/shibu.prasad/Desktop/FlutterStandardWithTPV/')
```

Comment:
@shibu000 can we remove these lines? Even though they are in the example project, it still will be visible to all users.